types/ws/index.d.ts

```diff
@@ -210,6 +210,7 @@ declare namespace WebSocket {
     interface ClientOptions extends SecureContextOptions {
         protocol?: string | undefined;
         followRedirects?: boolean | undefined;
+        generateMask?: (mask: Buffer) => void;
```

Comment:
Should this be:

```suggestion
        generateMask?(mask: Buffer): void;
```

To match the other function in this interfaces below / other interfaces?